src/components/MaterialTabs.js

```diff
@@ -172,8 +173,8 @@ export default class MaterialTabs extends React.Component<Props, State> {
             {this.props.items.map((item, idx) => (
               <Tab
                 allowFontScaling={this.props.allowFontScaling}
-                text={item}
-                key={item}
+                content={item}
+                key={idx}
```

Comment:
This is probably my only concern. Since persons used this library previous to change the number of tabs at runtime, this will have issues since you're using an index as a key.

Do we need to `idx` here if they're already passing in a key?

          items={[<Text key="1">Tab1</Text>, <Text key="2">Tab2</Text>]}

Reply:
Exactly!